decentraland/unity-renderer
# fix: [BiW Editor] Change rotation and position of the initial camera

## What does this PR change?


### Fixes #694
The initial camera while entering the editor makes no sense.
It needs to be changed, so it always looks at the scene from the South-West to the middle. Something like this:

**OLD:**
![image.png](https://images.zenhubusercontent.com/603cdb58edb53386a3d0d8d5/3fdb7fca-135c-4d6d-a9a3-d327e547fb27)

**NEW:**
![image](https://user-images.githubusercontent.com/64659061/123978292-083b8a00-d9c0-11eb-85a7-b219584e5f7d.png)

## How to test the changes?



1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:fix/BiW-change-rotation-and-position-of-the-initial-camera
2. Go to one of your owned lands.
3. Press K to open the BiW editor.
4. See the initial camera position (it should be the same indepent of the avatar position when you pressed K).
5. Move the camera along the scene.
6. Press `Shift + C` to reset the camera.
7. Notice the camera position is the same as initiallly.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
